app/Info.php

```diff
@@ -207,13 +207,15 @@ public function getPhoto($size = 'm')
 
     public function getDeviceIcon()
     {
+        if ($this->identities->contains('type', 'bot')
+        ||  $this->identities->contains('category', 'gateway')) {
+            return 'memory';
+        }
         if ($this->identities->contains('type', 'handheld')
-        || $this->identities->contains('type', 'phone')) {
+        || $this->identities->contains('type', 'phone')
+        || $this->identities->contains('type', 'sms')) {
             return 'smartphone';
         }
-        if ($this->identities->contains('type', 'bot')) {
-            return 'memory';
-        }
```

Comment:
Would it be possible to move those changes in a separate PR ? I'd prefer to keep things separated.